packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx

```diff
@@ -31,7 +31,7 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
     activeOption,
     clearSelection,
     getIndexOfId,
-    getOptionsMatchingValue,
+    getOptionsMatchingLabel,
```

Comment:
I changed this just so it wouldn't confuse me in the future; this isn't a public-facing function.